File: works/2309.java
import java.sql.Array;
import java.util.ArrayList;
import java.util.Collections;
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int[] arr = new int[10];
        int sum = 0;
        for(int i = 0; i < 9; i++)
        {
            arr[i] = sc.nextInt();
            sum += arr[i];
        }
        sum -= 100;

        int target1=0, target2=0;
        for(int i = 0; i < 9; i++)
        {
            for(int j = i+1; j < 9; j++)
            {
                if(arr[i] + arr[j] == sum)
                {
                    target1 = i; target2 = j;
                    break;
                }
            }
        }

        ArrayList<Integer> ans = new ArrayList<>();
        for(int i = 0; i < 9; i++)
        {
            if(i == target1 || i == target2) continue;
            ans.add(arr[i]);
        }
        Collections.sort(ans);

        for(int x : ans) System.out.println(x);
    }
}

File: works/2309.kt
import java.util.*
import kotlin.collections.ArrayList

fun main()
{
    val sc = Scanner(System.`in`)
    var arr = Array<Int>(10){0}
    var sum = 0

    for(i in 0 until 9)
    {
        arr[i] = sc.nextInt();
        sum += arr[i];
    }
    sum -= 100

    var target1 = 0; var target2 = 0
    for(i in 0 until 9)
    {
        for(j in 0 until 9)
        {
            if(arr[i] + arr[j] == sum)
            {
                target1 = i; target2 = j;
                break;
            }
        }
    }

    var ans = ArrayList<Int>()
    for(i in 0 until 9)
    {
        if(i == target1 || i == target2) continue
        ans.add(arr[i])
    }
    ans.sort()

    for(x in ans) println(x)
}